lib/ViewState/BasketState.php

```diff
@@ -229,6 +229,15 @@ public function vatInfoOk()
             )
         );
     }
+    
+    public function setVatRates( $vatRates )
+    {
+        // Sanity check the input
+        if( ! isset( $vatRates['rates']['AT']['standard_rate'] ) ){
+            throw new \Exception("VAT rates provided to the Basket appear to be invalid. No VAT rate found for Austria");
```

Comment:
was confused at first.. but austria is just an arbitrary sniff at the data I guess.